api/src/main/java/org/openmrs/api/impl/UserServiceImpl.java

```diff
@@ -719,4 +722,25 @@ public User saveUserProperties(Map<String, String> properties) {
 		return dao.saveUser(user, null);
 	}
 	
+	/**
+	 * @see UserService#changePassword(User, String, String)
+	 */
+	@Override
+	@Authorized(PrivilegeConstants.EDIT_USER_PASSWORDS)
+	@Logging(ignoredArgumentIndexes = 1)

```

Comment:
You need to ignore logging of both passwords, i see that you are only ignoring logging of the old password